h2quic/response_body.go

```diff
@@ -4,28 +4,15 @@ import (
 	"io"
 
 	quic "github.com/lucas-clemente/quic-go"
-	"github.com/lucas-clemente/quic-go/internal/utils"
 )
 
 type responseBody struct {
```

Comment:
You can probably drop the struct here.

Reply:
That doesn't work because `quic.Stream` is an interface.